diff --git a/node_modules/@codler/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js b/node_modules/@codler/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js
index 855c968..f8c5901 100644
--- a/node_modules/@codler/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js
+++ b/node_modules/@codler/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js
@@ -241,12 +241,24 @@ function KeyboardAwareHOC(
 
     scrollToPosition = (x: number, y: number, animated: boolean = true) => {
       const responder = this.getScrollResponder()
-      responder && responder.scrollResponderScrollTo({ x, y, animated })
+      if (responder) {
+        if (responder.scrollResponderScrollTo) {
+          responder.scrollResponderScrollTo({ x, y, animated })
+        } else if (responder.scrollTo) {
+          responder.scrollTo({ x, y, animated })
+        }
+      }
     }
 
     scrollToEnd = (animated?: boolean = true) => {
       const responder = this.getScrollResponder()
-      responder && responder.scrollResponderScrollToEnd({ animated })
+      if (responder) {
+        if (responder.scrollResponderScrollToEnd) {
+          responder.scrollResponderScrollToEnd({ animated })
+        } else if (responder.scrollToEnd) {
+          responder.scrollToEnd({ animated })
+        }
+      }
     }
 
     scrollForExtraHeightOnAndroid = (extraHeight: number) => {
